conf/modules.config

```diff
@@ -80,13 +80,14 @@ process {
         ]
     }
 
+    withName: WGET_GENOME {
```

Comment:
We tried wget two Years ago and we discarded this option. I am fine to have it but maybe we should avoid the pipeline to stop if one genome fails to be downloaded. Does maxRetries allow this? will the pipeline continue if the genome in not downloaded properly?

Reply:
since it is a module, there should be a way to skip it with errorStrategy 'ignore' and also having the output in the module set as optional: true

Reply:
Yes, but we can't set `errorStrategy` to both `retry` and `ignore` and I like the former. And we can't set the output in the module to optional, since that will skip the process _every time_.